### DISCUSSION TITLE
Migration v5 to v7

### DISCUSSION THREAD

**Discussion by stearruda** [storybookjs/storybook#22954] [Help]
Any suggestions on how to make a migration from version 5 to version 7?
I haven't found anything so easily on the community given that there's a significant jump of versions.

Should the migration go step by step (from 5 to 6) or directly (from 5 to 7)?

And expect what I should expect?

**Comment by f3bw** [comment]
Hello,

Storybook has a migration guide, here you can find known major breaking points. Nothing about moving from version 5 to version 7. 
https://storybook.js.org/docs/react/migration-guide

I would recommend creating a new branch and use the command: `npx storybook@latest upgrade`. If major problems arrive I would suggest another branch with the migration to version 6 first and then to version 7.
